risc0/zkvm/src/host/rpc.rs

```diff
@@ -50,6 +50,10 @@ pub struct ProofRequest {
 pub enum ShrinkWrapKind {
     /// TODO
     Groth16,
+
+    #[cfg(feature = "blake3")]
```

Comment:
since this is part of the on-the-wire encoding, I wouldn't conditionally compile this in, we should add it unconditionally and if we request it we can return an error

this is better than getting a can't decode request error